docs/index.asciidoc

```diff
@@ -144,17 +174,97 @@ specify a custom set of response headers
   * Value type is <<boolean,boolean>>
   * Default value is `false`
 
-SSL Configurations
+Events are by default sent in plain text. You can
+enable encryption by setting `ssl` to true and configuring
+the `ssl_certificate` and `ssl_key` options.
+
+[id="plugins-{type}s-{plugin}-ssl_certificate"]
+===== `ssl_certificate` 
+
+  * Value type is <<path,path>>
+  * There is no default value for this setting.
+
+SSL certificate to use.
+
+[id="plugins-{type}s-{plugin}-ssl_certificate_authorities"]
+===== `ssl_certificate_authorities` 
+
+  * Value type is <<array,array>>
+  * Default value is `[]`
+
+Validate client certificates against these authorities. 
+You can define multiple files or paths. All the certificates will
+be read and added to the trust store. You need to configure the `ssl_verify_mode`
+to `peer` or `force_peer` to enable the verification.
+
+
+[id="plugins-{type}s-{plugin}-ssl_handshake_timeout"]
+===== `ssl_handshake_timeout` 
 
-Enable SSL
+  * Value type is <<number,number>>
+  * Default value is `10000`
+
+Time in milliseconds for an incomplete ssl handshake to timeout
+
+[id="plugins-{type}s-{plugin}-ssl_key"]
+===== `ssl_key` 
+
+  * Value type is <<path,path>>
+  * There is no default value for this setting.
+
+SSL key to use.
```

Comment:
Oh ... is the PKCS#8 keystore ?

and `ssl_certificate` is the PEM variant ?  

..I am so confused.